addons/skin.confluence/720p/DialogVideoInfo.xml

```diff
@@ -955,6 +973,34 @@
 						<altlabel>1024</altlabel>
 						<usealttexture>Container.Content(TVShows)</usealttexture>
 					</control>
+					<control type="button" id="666">
+						<description>Decrease Rating</description>
+						<left>160</left>
+						<top>5</top>
+						<width>33</width>
+						<height>22</height>
+						<onclick>DecreaseRating</onclick>
+						<texturenofocus>scroll-down-2.png</texturenofocus>
+						<texturefocus>scroll-down-focus-2.png</texturefocus>
+						<onleft>8</onleft>
```

Comment:
I think you dont need these navigation nodes (onleft/right/up/down) since that should be managed by the parent grouplist.